internal/pkg/template/templates/workloads/partials/cf/https-listener.yml

```diff
@@ -45,43 +45,44 @@ LoadBalancerDNSAlias{{$hostedZoneID}}:
 {{- end}}
 {{- end}}
 
-HTTPSRulePriorityAction:
+{{- range $i, $listener := .ALB.Rules}}
+HTTPSRulePriorityAction{{ if ne $i 0 }}{{ $i }}{{ end }}:
```

Comment:
Let's discuss offline, but my recommendation is to get this PR unblocked since no customer can leverage multi-rules yet.

We can start with approach (1) that should work but is costly and do a quick follow-up with a long-term proper fix.

Reply:
We are taking care of this issue in the fix [here](https://github.com/aws/copilot-cli/pull/4568).